src/identity/reducers/index.test.ts

```diff
@@ -0,0 +1,67 @@
+// Reducer
+import reducer from 'src/identity/reducers'
+import {initialState} from 'src/identity/reducers'
+
+// Actions
+import {
+  setQuartzIdentity,
+  setCurrentBillingProvider,
+  setCurrentOrgDetails,
+} from 'src/identity/actions/creators'
+
+// Mocks
+import {
+  mockIdentities,
+  mockBillingProviders,
+  mockOrgDetailsArr,
+} from 'src/identity/mockUserData'
+
+// Utils
+import {omit} from 'lodash'
+
+describe('identity reducer', () => {
+  it('can initialize a default state', () => {
+    const newState = reducer(undefined, setQuartzIdentity(initialState))
+    expect(newState).toEqual(initialState)
+  })
+
+  it('can change the user identity using quartzIdentity', () => {
+    for (let i = 0; i < mockIdentities.length; i++) {
+      const expectedState = mockIdentities[i]
+      const actual = reducer(undefined, setQuartzIdentity(mockIdentities[i]))
+      expect(actual).toEqual(expectedState)
+    }
```

Comment:
these days, I'm in favor of delegating the looping logic to `forEach` for simple iteration cases like this, or `map` when you need to construct an array. 

```ts
mockIdentities.forEach(mockIdentity => {
  expect(reducer(undefined, setQuartzIdentity(mockIdentity[i])))
  .toEqual(mockIdentity)
})

Reply:
Agree, implemented. Not sure why I wasn't doing that in the test file, since there's no need to exit the loop early.